src/test/java/teammates/sqllogic/core/FeedbackQuestionsLogicTest.java

```diff
@@ -47,16 +41,23 @@ public void setUpMethod() {
         fqLogic.initLogicDependencies(fqDb, coursesLogic, frLogic, usersLogic, feedbackSessionsLogic);
     }
 
-    @Test(enabled = false)
+    @Test
     public void testGetFeedbackQuestionsForSession_questionNumbersInOrder_success() {
-        FeedbackSession fs = typicalDataBundle.feedbackSessions.get("session1InCourse1");
-        FeedbackQuestion fq1 = typicalDataBundle.feedbackQuestions.get("qn1InSession1InCourse1");
-        FeedbackQuestion fq2 = typicalDataBundle.feedbackQuestions.get("qn2InSession1InCourse1");
-        FeedbackQuestion fq3 = typicalDataBundle.feedbackQuestions.get("qn3InSession1InCourse1");
-        FeedbackQuestion fq4 = typicalDataBundle.feedbackQuestions.get("qn4InSession1InCourse1");
-        FeedbackQuestion fq5 = typicalDataBundle.feedbackQuestions.get("qn5InSession1InCourse1");
-
-        List<FeedbackQuestion> questions = List.of(fq1, fq2, fq3, fq4, fq5);
+        Course c = getTypicalCourse();
+        FeedbackSession fs = getTypicalFeedbackSessionForCourse(c);
+        FeedbackQuestion fq1 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq2 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq3 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq4 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq5 = getTypicalFeedbackQuestionForSession(fs);
+
+        fq1.setQuestionNumber(1);
+        fq2.setQuestionNumber(2);
+        fq3.setQuestionNumber(3);
+        fq4.setQuestionNumber(4);
+        fq5.setQuestionNumber(5);
+
+        ArrayList<FeedbackQuestion> questions = new ArrayList<>(List.of(fq2, fq4, fq3, fq1, fq5));
```

Comment:
For better code quality, how about using a for loop e.g.:
```suggestion
        List<FeedbackQuestion> questions = new ArrayList<>();
        for (int i = 1; i <= 5; i++) {
            FeedbackQuestion fq = getTypicalFeedbackQuestionForSession(fs);
            fq.setQuestionNumber(i);
            questions.add(fq);
        }
```

Reply:
@cedricongjh does the insertion order matter (e.g. 2-4-3-1-5 here and below)? Seems that it shouldn't since the previous order was 1-2-3-4-5

Reply:
the order doesn't matter here, but does matter in the test below as the test below `testGetFeedbackQuestionsForSession_questionNumbersOutOfOrder_success` is testing for when the questions returned by the db layer is out of order